React/Views/RCTView.m

```diff
@@ -176,6 +177,7 @@ - (instancetype)initWithFrame:(CGRect)frame
 #if TARGET_OS_OSX // [macOS
     _transform3D = CATransform3DIdentity;
     _shadowColor = nil;
+    _mouseDownCanMoveWindow = YES;
```

Comment:
feels like it should be no by default?

Reply:
https://developer.apple.com/documentation/appkit/nsview/1483666-mousedowncanmovewindow
> The default value of this property is NO if the view is opaque; otherwise, it is set to YES. Subclasses can override this property to return different values based on the event.

Maybe we can mimic that. I'll think about how to properly maybe inherit the value.

Reply:
I don't think I can do this. Opaque is `NO` by default as of https://github.com/microsoft/react-native-macos/pull/1612 . I found more info about the implementation relationship here: https://gist.github.com/bjhomer/2a0035fa516dd8672fe7

Reading that had me think we should leave it default YES. If it is default NO, then it basically is never on and you can never drag the window.